cmd/kubelet/app/server.go

```diff
@@ -953,33 +952,6 @@ func RunKubelet(kubeServer *options.KubeletServer, kubeDeps *kubelet.Dependencie
 	// Setup event recorder if required.
 	makeEventRecorder(kubeDeps, nodeName)
 
-	// TODO(mtaufen): I moved the validation of these fields here, from UnsecuredKubeletConfig,
-	//                so that I could remove the associated fields from KubeletConfiginternal. I would
-	//                prefer this to be done as part of an independent validation step on the
-	//                KubeletConfiguration. But as far as I can tell, we don't have an explicit
-	//                place for validation of the KubeletConfiguration yet.
-	hostNetworkSources, err := kubetypes.GetValidatedSources(kubeServer.HostNetworkSources)
-	if err != nil {
-		return err
-	}
-
-	hostPIDSources, err := kubetypes.GetValidatedSources(kubeServer.HostPIDSources)
-	if err != nil {
-		return err
-	}
-
-	hostIPCSources, err := kubetypes.GetValidatedSources(kubeServer.HostIPCSources)
-	if err != nil {
-		return err
-	}
-
-	privilegedSources := capabilities.PrivilegedSources{
-		HostNetworkSources: hostNetworkSources,
-		HostPIDSources:     hostPIDSources,
-		HostIPCSources:     hostIPCSources,
-	}
-	capabilities.Setup(kubeServer.AllowPrivileged, privilegedSources, 0)
```

Comment:
What about `perConnectionBytesPerSec` (the third argument to `capabilities.Setup`): do we still want to explicitly configure this to 0?

Can you also remove the `AllowPrivileged` and `PrivilegedSources` related functionality from `pkg/capabilities/capabilities.go`?

Reply:
@mtaufen Do you mean `capabilities.Setup()` should be preserved?  
`AllowPrivileged` and `PrivilegedSources` in `capabilities.go` are removed.

Reply:
Yes, I was wondering if it would cause any issues to leave it out, because it also initializes a `nil` pointer in the global `capInstance` struct.

Reply:
It's probably safe, now that I take another look, since `Get` checks for that case and still assigns 0 as a default, so I'll lgtm this.